src/Models/ToolbarOptions.ts

```diff
@@ -18,5 +18,11 @@ export default class ToolbarOptions {
        this.topPager = true;
        this.rowsPerPageOptions = [10, 20, 50, 100];
        this.itemsPerPage = 10;
+
+       if (options){
```

Comment:
This change failed for the others grid cases. This change set all the options as false when the options aren't declared it is wrong.